src/requests/session.ts

```diff
@@ -14,11 +15,11 @@ export class SessionClient {
     this.client = client;
   }
 
-  async find(sessionId: string): Promise<Session> {
+  async find(sessionId: string): Promise<Session | null> {
```

Comment:
I'm fixing some of these type defs where Hasura can return `null` when the resource isn't found.